README.md

```diff
@@ -1 +1,8 @@
-# my-sms
\ No newline at end of file
+# my-sms
+## VSCode
+Install docker desktop, compose and the vscode remote development extension (see https://code.visualstudio.com/docs/remote/containers).
+Remote development config is stored in .devcontainer and will set up automatically just by opening the folder in vscode.
+Once container is set up rails server will autostart and vscode will forward the internal port to one available.
+
+## Manual
+Run docker compose file then install bundle and run rails server from my-sms directory.
```

Comment:
Can you provide actual commands on getting this running? It's nice to have a reference in the README.
Maybe something along the lines of

```
## Development

Use [Docker](https://docs.docker.com/docker-for-mac/install/) to run in containers.

Once Docker is installed on your system, you can use the following commands from the root of the application folder, e.g `/var/www/my-sms`:

### Build images:

`docker-compose build`

### Run containers in background:

`docker-compose up -d`
Open `...` in your browser.

### Bundle install:

`docker-compose run --rm app bundle install`